testpar/t_bigio.c

```diff
@@ -26,7 +26,7 @@ const char *FILENAME[3] = {"bigio_test.h5", "single_rank_independent_io.h5", NUL
 #define DATASET4             "DSET4"
 #define DXFER_COLLECTIVE_IO  0x1 /* Collective IO*/
 #define DXFER_INDEPENDENT_IO 0x2 /* Independent IO collectively */
-#define DXFER_BIGCOUNT       (1 < 29)
```

Comment:
This seems to have been overlooked, but I'm also a bit hesitant about this change because it seems the original intention was to have the bigio test creating datasets with 500M elements in one of their two dimensions, which quickly locks my machine up when testing. In the VOL tests I've adjusted this value to 1310720 to be a little more reasonable, but I think the test needs reviewing and/or rewriting as a whole.

Reply:
Agreed, we probably need a reasonable value depending on the resources available.

Reply:
Yes, @lrknox we should probably hold off on merging this until we decide a reasonable value for this test. It takes a long long time after the above is correctly fixed.